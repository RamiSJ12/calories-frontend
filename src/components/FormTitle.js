import { Typography } from '@material-ui/core';
import React from 'react';
import PropTypes from 'prop-types';
import commonStyles from '../utils/commonStyles';

const FormTitle = ({ title }) => {
  const commonClasses = commonStyles();
  return (
    <Typography
      variant="h2"
      className={`${commonClasses.textSecondary} ${commonClasses.centerText} ${commonClasses.topMargin30}`}
    >
      {title}
    </Typography>
  );
};

FormTitle.propTypes = {
  title: PropTypes.string.isRequired,
};

export default FormTitle;
