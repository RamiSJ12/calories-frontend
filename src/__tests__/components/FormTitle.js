import renderer from 'react-test-renderer';
import FormTitle from '../../components/FormTitle';

describe('FormTitle component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<FormTitle title="Home" />);
    expect(tree).toMatchSnapshot();
  });
});
