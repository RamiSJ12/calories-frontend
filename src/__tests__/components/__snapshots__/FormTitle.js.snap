// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTitle component should match snapshot 1`] = `
<h2
  className="MuiTypography-root makeStyles-textSecondary-2 makeStyles-centerText-5 makeStyles-topMargin30-11 MuiTypography-h2"
>
  Home
</h2>
`;
